sonic-net/sonic-buildimage
# Allow TSA on ibgp sessions between linecards on packet chassis

#### Why I did it
On packet chassis, ibgp sessions between ASICs of different linecards are also considered internal. Currently, these internal ibgp sessions are skipped while the device is isolated with TSA. As a result, when TSA is done on one linecard, bgp routes are still advertised to other linecards, hence drawing traffic (if the other linecard is not isolated).

#### How I did it
For packet chassis, allow TSA to withdraw routes from internal ibgp sessions as well. As a result of this change, the ibgp sessions between ASICs on the same linecard are also isolated when TSA is done.

This change does not impact the current behavior on multi-asic pizza boxes, where the ibgp sessions between the asics are not isolated on TSA.

#### How to verify it
* Run TSA on a specific linecard (LC1) of packet chassis
* Check routes advertised to other linecards from LC1 and ensure only loopback is advertised

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog


#### Ensure to add label/tag for the feature raised. example - PR#2174 under sonic-utilities repo. where, Generic Config and Update feature has been labelled as GCU.

#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
